### PR TITLE
Feature/updating contact through post api

### PR DESCRIPTION
**Pull request by Pjaijai**
Mainly adding the post url and user profile url. The email will be more cleaner

### CODE REVIEW

File: supabase/functions/contact-through-post/index.ts [hunk 1]
```diff
@@ -86,31 +86,23 @@ serve(async (req: any) => {
...
-    }有興趣`
+    const subject = `${sender.username}對於你份街招有興趣`
     const body = `
             <html lang="zh-Hk">
             <body>
                 <p>Hi ${post.user.username}!</p>
-                <p>${sender.username}對於你你份街招${
-      post.type === "referer" ? "(工搵人)" : "(人搵工)"
-    }有興趣</p>
+                <p>${sender.username}對於你你份街招有興趣</p>
```
Comment (yaucp) [inline supabase/functions/contact-through-post/index.ts:94]: 你 double jor

Comment (Pjaijai) [reply supabase/functions/contact-through-post/index.ts]: updated

**Review by yaucp — DISMISSED** [review]

File: supabase/functions/contact-through-post/index.ts [hunk 2]
```diff
@@ -86,31 +86,23 @@ serve(async (req: any) => {
...
-    }有興趣`
+    const subject = `${sender.username}對於你份街招有興趣`
     const body = `
             <html lang="zh-Hk">
             <body>
                 <p>Hi ${post.user.username}!</p>
-                <p>${sender.username}對於你你份街招${
-      post.type === "referer" ? "(工搵人)" : "(人搵工)"
-    }有興趣</p>
+                <p>${sender.username}對於你份街招有興趣</p>
```
Comment (yaucp) [inline supabase/functions/contact-through-post/index.ts:94]: 其實「對你份街招有興趣」好似自然啲

Comment (Pjaijai) [reply supabase/functions/contact-through-post/index.ts]: updated

**Review by yaucp — APPROVED** [review]

**Pull request merged**